build.gradle

```diff
@@ -55,7 +55,7 @@ ext {
 	 * Versions not necessarily bumped by a script (testing, etc...) below:
 	 */
 	// Misc not often upgraded
-	jsr305Version = '3.0.2'
+	findbugsVersion = '3.0.1'
```

Comment:
3.0.1 is the last version that actually has the `annotations` jar. Downgrading to 3.0.1 doesn't seem to be a problem.

Reply:
can you clarify what you mean by "`annotations` jar"? I do see `jsr305-3.0.2.jar` on [Maven Central](https://search.maven.org/artifact/com.google.code.findbugs/jsr305/3.0.2/jar)

also, the version is very specifically for jsr305, not findbugs, so I wouldn't rename the variable.

Reply:
`annotations` is a __different__ findbugs jar, which contains [`@SuppressFBWarnings`](https://stackoverflow.com/questions/39412365/what-to-import-to-use-suppressfbwarnings). And this is what makes me think that the version is __not__ the version of jsr305, but the version of findbugs, which happens to re-distribute jsr305. But it is indeed the version of findbugs, globally.
Sadly, 3.0.1 is the last version that ships the annotations.jar
Hope this makes sense.

Reply:
ok so this is needed for a dependency in `reactor-tool` so that ByteBuddy doesn't warn about missing annotations (including `SuppressFBWarnings`)